src/main/scala/tech/cryptonomic/conseil/util/DatabaseUtil.scala

```diff
@@ -103,11 +103,12 @@ object DatabaseUtil {
         * @return new SQLActionBuilder containing limit statement
         */
       def addGroupBy(aggregation: List[Aggregation], columns: List[String]): SQLActionBuilder = {
-        if(aggregation.isEmpty) {
+        val aggregationFields = aggregation.map(_.field).toSet
+        val existingAggregationFields = columns.toSet.diff(aggregationFields).toList
```

Comment:
I'm not sure if the name express what is in this set

Reply:
renamed